sopel/modules/url.py

```diff
@@ -54,6 +66,14 @@ def configure(config):
         ' shorter version of the URL in the title as a TinyURL link'
         ' (0 to disable)'
     )
+    config.url.configure_setting(
+        'enable_private_resolution',
+        'Enables URL lookups for RFC1918 addresses'
```

Comment:
I've noticed other `boolean` settings in the config wizard phrase these as questions, with an explicit `?` at the end (e.g., `admin.py`). What do you think about changing this prompt (and the next) to something like `Enable URL lookups for RFC1918 address?` (remove extra `s`, and add that `?` in)